website/blog/2023-04-18-aluminium.md

```diff
@@ -0,0 +1,391 @@
+---
+author: Katarzyna Marek
+title: Metals v0.11.12 - Aluminium
+authorURL: https://github.com/kasiaMarek
+authorImageURL: https://github.com/kasiaMarek.png
+---
+
+We're happy to announce the release of Metals v0.11.12, which brings mostly stability fixes and some improvements. It also starts an effort towards improving identification of issues for users.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">100</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">80</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">20</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/56?closed=1]
+(https://github.com/scalameta/metals/milestone/56?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Error Reports](#introduce-error-reports)
+- [Code action for converting dependencies from sbt to mill style](#code-action-for-converting-dependencies-from-sbt-to-mill-style)
+- [Improvements to semantic highlighting](#improvements-to-semantic-highlighting)
+- [Miscellaneous](#miscellaneous)
+
+## Error Reports
+
+Starting with this release, metals will automatically generate error reports, which can later be used by users to attach to github issue for additional information. Upon any error or incorrect state metals will create an error report in `.metals/.reports` directory within the workspace. All the reports have their paths anonymised.
```

Comment:
What do you think:

```suggestion
Starting with this release, metals will automatically generate error reports when metals when something goes wrong with metals, which can later be used by users to attach to github issue for additional information. Upon any error or incorrect state metals will create an error report in `.metals/.reports` directory within the workspace. All the reports have their paths anonymised.
```

? 

At first glance of "metals will automatically generate error reports", I thought "wait, Metals will collect the compilation error or something in my project?" (I know that's not the case though 😅 )

Reply:
Will writing explicitly **in metals** be enough?
```
Upon any error or incorrect state in metals an error...
```

Reply:
> Will writing explicitly in metals be enough?

Yeah, I think that's enough :+1: but I think "in metals" should come at the first sentence.

Reply:
> Starting with this release, upon chosen errors or incorrect states in metals error reports will be automatically generated and saved in the `.metals/.reports` directory within the workspace

Looks good 💯